src/string_bitflip.cc

```diff
@@ -0,0 +1,20 @@
+#include "string_bitflip.h"
+
+#include "node.h"
+#include "v8.h"
+
+#include <string.h>
+#include <limits.h>
+
+namespace node {
+
+void StringBitflip::BE_FlipBits (uint16_t* dst,
+                                 const uint16_t* src,
+                                 size_t buflen) {
+
+  for (size_t i = 0; i < buflen; i++) {
+    dst[i] = (src[i] << 8) | (src[i] >> 8);
+  }
+
+};
```

Comment:
Style: no ; and no blank lines in the function body.  You should put a blank line _after_ the function body however. :-)